src/task.rs

```diff
@@ -2407,3 +2393,14 @@ impl<'a> Runner<'a> {
         Ok(all_ok)
     }
 }
+
+pub fn relative_path(name: &String, cwd: &String) -> String {
```

Comment:
Better to use an actual string reference I think:

```suggestion
pub fn relative_path(name: &str, cwd: &str) -> String {
```

See eg https://www.ameyalokare.com/rust/2017/10/12/rust-str-vs-String.html